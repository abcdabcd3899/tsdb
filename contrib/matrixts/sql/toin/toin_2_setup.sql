set search_path to test_toin;

--
-- fill the settings for the template
--

\set tname 't_data'
\set iname 'i_data'
\set itype 'toin'
\set keys 'c1, c2'
\set opts 'with (pages_per_range=1)'

--
-- run the template
--

\i sql/toin/operator_setup.template
