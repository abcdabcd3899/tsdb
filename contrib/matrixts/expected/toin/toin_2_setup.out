set search_path to test_toin;
--
-- fill the settings for the template
--
\set tname 't_data'
\set iname 'i_data'
\set itype 'toin'
\set keys 'c1, c2'
\set opts 'with (pages_per_range=1)'
--
-- run the template
--
\i sql/toin/operator_setup.template
--
-- cleanup
--
-- start_ignore
drop index if exists :iname;
-- end_ignore
--
-- build index
--
create index :iname on :tname using :itype (:keys) :opts;
